src/generator/__snapshots__/errata.integration.js.snap

```diff
@@ -714,7 +714,7 @@ exports[`generates a simple playbook with single RHSA remediation 1`] = `
   become: true
   tasks:
     - name: check for update
-      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q --advisory {{ insights_issues | regex_search('RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5}( --advisory RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5})*') }}\\"
+      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q {{ insights_issues | regex_search('(--advisory (RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5}))+') }}\\"
```

Comment:
I believe that the `advisory` REGEX has to be changed as well in the same manner as the `cve`

--> `([ ]*--advisory (RH[SBE]A-20[\d]{2}:[\d]{4,5}))+`